src/mcMd/mdIntegrators/NveVvIntegrator.cpp

```diff
@@ -142,8 +142,8 @@ namespace McMd
          }
       }
       #endif
-      system().positionSignal().notify();
-      system().velocitySignal().notify();
+      //system().positionSignal().notify();
```

Comment:
Please explain why it was necessary to comment out these two system().positionSignal().notify() and 
system().velocitySignal().notify() commands in lines 145:146 of NveVvIntegrator.cpp. What error occurred
if you did not do this?